q2_demux/_format.py

```diff
@@ -18,6 +18,17 @@ class EMPMultiplexedDirFmt(model.DirectoryFormat):
         r'barcodes.fastq.gz', format=FastqGzFormat)
 
 
+class PairedEMPMultiplexedDirFmt(model.DirectoryFormat):
```

Comment:
I'd be open to dealing with now. This name is my least favorite of all of the names.